app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt

```diff
@@ -32,11 +32,14 @@ class WallpaperManager(
     private val settings: Settings,
     private val downloader: WallpaperDownloader,
     private val fileManager: WallpaperFileManager,
+    private val currentLocale: String,
     allWallpapers: List<Wallpaper> = availableWallpapers
 ) {
     val logger = Logger("WallpaperManager")
 
-    val wallpapers = allWallpapers.filter(::filterExpiredRemoteWallpapers)
```

Comment:
I will just introduce another filter here to reduce the risk of missing something close to prime time 😅